.github/workflows/deploy.yaml

```diff
@@ -8,12 +8,12 @@ permissions:
 jobs:
```

Comment:
ATM It's better to disable this job because it's too early to deploy on the main website.

@Takeno mentioned offline that there are a few texts to update.

Reply:
Sure 👍

You can push changes you want to this PR, if you want, or you can start a review.

Reply:
Done